examples/secured-service-with-jwt/secured_service_with_jwt.bal

```diff
@@ -39,13 +39,13 @@ endpoint http:SecureListener ep {
     }
 }
 // Auth configuration comprises of two parts - authentication & authorization.
-// Authentication can be enabled by setting 'authentication:{enabled:true}'
-// annotation attribute.
+// Authentication can be enabled by setting `authentication:{enabled:true}`
```

Comment:
setting -> setting the
?

Reply:
Fixed with commit https://github.com/ballerina-platform/ballerina-examples/pull/760/commits/2f92b131b2a834624c5caec417defcf4c2a6096f